cpu/cortexm_common/include/vendor/avr/pgmspace.h

```diff
@@ -0,0 +1,122 @@
+/*
```

Comment:
Since such a module would only have a header file (`pgmspace.h`) would it be a _pseudomodule_?

My first try isn't working, is there an example of a header-only module? (I only found `sys/libc` but this does not have any Makefile).

Reply:
If there is no Makefile, it has to be a pseudo module

Reply:
These files are gone now from the source tree, since they're included in the new `arduino_api` package.